content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md

```diff
@@ -358,8 +358,8 @@ Readiness probes runs on the container during its whole lifecycle.
 {{< /note >}}
 
 {{< caution >}}
-Liveness probes *do not* wait for readiness probes to succeed.
-If you want to wait before executing a liveness probe you should use
+The readiness probe does not wait for the liveness probe to succeed.
```

Comment:
Maybe add something like "The readiness probe and the liveness probe do not have dependencies on each other" would be better.

Reply:
Hi @tengqm Changes made as per the suggestion PTAL!